packages/utilities/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './Async';
 export * from './AutoScroll';
 export * from './BaseComponent';
+export * from './Context';
```

Comment:
One thought might be;

Don't export from the index. Import the lib/Context file path to avoid the bundle impact in cases where people aren't using resize group.

Webpack tree shaking would help tremendously here, but I think we're still waiting on this.